src/components/IWantUrNFTForm.vue

```diff
@@ -255,6 +298,11 @@ export default defineComponent({
           isLoading.value = false;
         });
 
+        // Email question asker about their question being answered
+        sendEmailUpdateQuestionAsker();
+
+        // Email question responder about their answer
+        sendEmailUpdateQuestionResponder();
```

Comment:
can we comment this out (for now)? I'm not convinced we want to send an email to the person who answers the question atm.